applications/crossbar/src/crossbar_maintenance.erl

```diff
@@ -435,13 +497,13 @@ validate_account(JObj, Context) ->
                                   ,{fun cb_context:set_resp_status/2, 'fatal'}
                                   ])
               ],
-    Context1 = crossbar_bindings:fold(<<"v1_resource.validate.accounts">>, Payload),
+    Context1 = crossbar_bindings:fold(<<"v2_resource.validate.accounts">>, Payload),
```

Comment:
There's no cb_accounts_v2 though, right? This does not actually change anything or am I missing something?